hadoop-common-project/hadoop-common/src/site/markdown/filesystem/bulkdelete.md

```diff
@@ -161,124 +105,20 @@ store.hasPathCapability(path, "fs.capability.bulk.delete")
 
 ### Invocation through Reflection.
 
-The need for many Libraries to compile against very old versions of Hadoop
+The need for many libraries to compile against very old versions of Hadoop
 means that most of the cloud-first Filesystem API calls cannot be used except
 through reflection -And the more complicated The API and its data types are,
 The harder that reflection is to implement.
 
-To assist this, the class `org.apache.hadoop.fs.FileUtil` has two methods
+To assist this, the class `org.apache.hadoop.io.wrappedio.WrappedIO` has few methods
 which are intended to provide simple access to the API, especially
 through reflection.
 
 ```java
-  /**
-   * Get the maximum number of objects/files to delete in a single request.
-   * @param fs filesystem
-   * @param path path to delete under.
-   * @return a number greater than or equal to zero.
-   * @throws UnsupportedOperationException bulk delete under that path is not supported.
-   * @throws IllegalArgumentException path not valid.
-   * @throws IOException problems resolving paths
-   */
+
   public static int bulkDeletePageSize(FileSystem fs, Path path) throws IOException;
   
-  /**
-   * Delete a list of files/objects.
-   * <ul>
-   *   <li>Files must be under the path provided in {@code base}.</li>
-   *   <li>The size of the list must be equal to or less than the page size.</li>
-   *   <li>Directories are not supported; the outcome of attempting to delete
-   *       directories is undefined (ignored; undetected, listed as failures...).</li>
-   *   <li>The operation is not atomic.</li>
-   *   <li>The operation is treated as idempotent: network failures may
-   *        trigger resubmission of the request -any new objects created under a
-   *        path in the list may then be deleted.</li>
-   *    <li>There is no guarantee that any parent directories exist after this call.
-   *    </li>
-   * </ul>
-   * @param fs filesystem
-   * @param base path to delete under.
-   * @param paths list of paths which must be absolute and under the base path.
-   * @return a list of all the paths which couldn't be deleted for a reason other than "not found" and any associated error message.
-   * @throws UnsupportedOperationException bulk delete under that path is not supported.
-   * @throws IOException IO problems including networking, authentication and more.
-   * @throws IllegalArgumentException if a path argument is invalid.
-   */
-  public static List<Map.Entry<Path, String>> bulkDelete(FileSystem fs, Path base, List<Path> paths)
-```
+  public static int bulkDeletePageSize(FileSystem fs, Path path) throws IOException;
 
-## S3A Implementation
-
-The S3A client exports this API.
```

Comment:
this needs to be covered, along with the default implementation "maps to delete(path, false)"